src/v2/Components/FlashBanner.tsx

```diff
@@ -25,7 +22,22 @@ const messages: Record<FlashMessageKey, string | React.FC> = {
   expired_token: "Link expired. Resend verification email.",
```

Comment:
Related to question below -- if we can avoid `snake_case` that would be best, unless these properties directly interface with code managed in gravity (or some other ruby codebase). A lot of the work in MP v2 was to convert snake to camel, and therefore bring a greater consistency across all new JS code.